restli-tools/src/test/pegasus/Bar.pdl

```diff
@@ -0,0 +1,3 @@
+record Bar {
```

Comment:
What is the purpose of the following test schema doing?
It is neither in invalid or valid directory.

Reply:
This is base schema which is included in the extension schema. I will add comment doc.